pkg/translate/translateValue_test.go

```diff
@@ -225,7 +225,7 @@ mixer:
 	}
 	tr, err := NewValueYAMLTranslator(version.NewMinorVersion(1, 2))
 	if err != nil {
-		t.Fatal("fail to get value.yaml translator")
+		t.Fatal("fail to get helm value.yaml translator")
```

Comment:
General comment about this UT file - I recommend putting the input before the output in the test struct ordering. 
Also, now that implementation is close to the final, we need more complete UT coverage (e.g. the k8s stuff, namespaces, enablement etc.)

Reply:
done